packages/crypto/src/address/encoding/base58.ts

```diff
@@ -0,0 +1,14 @@
+import { base58 } from "@scure/base"
+
+export const encodeAddressBase58 = (publicKey: Uint8Array): string => {
+  return base58.encode(publicKey)
+}
+
```

Comment:
Adds the NOTE to the docstring for the function

```suggestion
/** Detect if address is base58 encoded (NOTE: also returns true for ss58 addresses) */
```